packaging/android/build_and_publish.sh

```diff
@@ -36,6 +40,14 @@ function reset {
 }
 trap reset EXIT
 
+if [ -n "$kbfs_commit" ]; then
+  cd "$kbfs_dir"
+  echo "Checking out $kbfs_commit on client (will reset to $kbfs_branch)"
+  git checkout "$kbfs_commit"
+  # tell gobuild.sh to use our local commit
+  export LOCAL_KBFS=1
```

Comment:
Where exactly is `gobuild.sh` called? If it's called by the caller of this script, then I don't think this works; see https://stackoverflow.com/questions/16618071/can-i-export-a-variable-to-the-environment-from-a-bash-script-without-sourcing-i

Reply:
this script does it when it calls `  CHECK_CI="$check_ci" yarn run rn-gobuild-ios
`